discovery-provider/src/queries/queries.py

```diff
@@ -569,3 +570,22 @@ def get_ipfs_peer_info_route():
         return api_helpers.success_response(ipfs_peer_info)
     except exceptions.ArgumentError as e:
         return api_helpers.error_response(str(e), 400)
+
+@bp.route("/get_sol_play", methods=("GET",))
```

Comment:
We have moved over routes into the `src/api/v1/<ENTITY>.py` files 
I'm not sure where these might fit in, in the context of existing route structure. 
thoughts @raymondjacobson

Reply:
not totally familiar w/ the context in which we'd call these, but keeping them out of queries.py would be nice if we want these to live long term

Reply:
i wanted to expose this just so we have the information available but now on second thought - is there a better way you'd recommend to verify each play individually in a test scenario? because that's really what `get_sol_play` is for - the `milestones` is to incorporate sol plays in the notification sending